test.dockerfile

```diff
@@ -14,3 +14,15 @@ RUN test $(which dune) = "/usr/bin/dune"
 
 # Test that the installed dune can be executed and that it reports being the expected version:
 RUN test $(dune --version) = "$DUNE_VERSION"
+
+# Start from a fresh layer, and this time install dune having the script first
+# look up the latest released version of dune.
+FROM alpine:3.22.0
+RUN apk update && apk add curl git
+COPY install.sh .
+
+# Install dune system-wide using the install script:
+RUN ./install.sh --install-root /usr --no-update-shell-config
+
+# Test that dune was installed to the expected location:
+RUN test $(which dune) = "/usr/bin/dune"
```

Comment:
Maybe also helps checking which version of Dune was installed? But that also means the version needs to be bumped every time a new version is released? I don't know what's a future proof way to add it.

Reply:
> But that also means the version needs to be bumped every time a new version is released?

Yep that's why I left the version out of this test. Taking actions outside of this repository (pushing a new tag to https://github.com/ocaml-dune/dune-bin) would cause this test to start failing. I think it's ok to leave the version number out of this test.